components/finance/ExpenseButton/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, createStyles, makeStyles } from '@material-ui/core'
+import React, { useState } from 'react'
+import { useToggle } from 'react-use'
+
+import ClientSideRendering from '../../../lib/client-side-rendering'
+import ExpensePanelDetails from '../ExpensePanelDetails'
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    deleteButton: {
+      color: 'white',
+      background: '#f16c6c',
+      '&:hover': {
+        background: '#f16c6c',
+      },
+    },
+  })
+)
+
+export default function ExpenseButton() {
+  const classes = useStyles()
+  const [isOn, toggleIsOn] = useToggle(false)
+  const [amount, updateAmountSum] = useState(0.0)
+  const currencyList = ['INR', '$', 'YUH']
```

Comment:
the list of currency needs to be fetched from https://www.wallora.com/api/currencies
Add a graphql endpoint to return this list of currencies